karma.conf.js

```diff
@@ -45,7 +45,7 @@ module.exports = function (config) {
         port: 9669,
         colors: true,
         autoWatch: true,
-        singleRun: false,
+        singleRun: true,
```

Comment:
@Simply007 can you please bring back the "karma-travis.conf.js"? Maybe rename it like karma-gh.conf.js. I'm using the 'singleRun' only for CI, but for development I want to keep the tests running so I can make dynamic changes to code and have tests run automatically ;)

Reply:
Ok

Reply:
77f5c6b